migrations/env.py

```diff
@@ -2,13 +2,15 @@
 from logging.config import fileConfig
 
 from alembic import context
-from sqlalchemy import engine_from_config, pool
 
 from db.schemas import BaseORM
+from renku_crac.config import Config as CracConfig
 
 # this is the Alembic Config object, which provides
 # access to the values within the .ini file in use.
 config = context.config
+crac_config = CracConfig.from_env()
+# config.set_section_option("alembic", "sqlalchemy.url", crac_config.db.sql_alchemy_url)
```

Comment:
No that is not needed. I just forgot to remove it when I was trying to figure this out. You can see that in subsequent lines in all cases where migrations are done we are using the sql connection from our own config (i.e. `CracConfig`) not from alembic. So that can be safely removed.